src/DoubleLinkedList.java

```diff
@@ -0,0 +1,68 @@
+public class DoubleLinkedList {
+
+    Node head;
+
+    public void insertHead(int date){
+        Node newNode = new Node(date);
+        newNode.next = head;
+        if (head != null) {
+            head.prev = newNode;
+        }
+        head = newNode;
+    }
+
+    public void insertTail(int date){
+        Node newNode = new Node(date);
+        Node tail = head;
+
+        newNode.next= null;
+
+        if (head==null) {
+            newNode.prev = null;
+            head = newNode;
+            return;
+        }
+        while (tail.next != null){
+            tail = tail.next;
+        }
+        tail.next= newNode;
+        newNode.prev = tail;
+    }
+
+    public void deleteNode(Node delete){
```

Comment:
This function could receive then value as parameter, as the other functions for consistency.